### PR TITLE
docs: Tutorial for getting started

### PR DESCRIPTION
**Pull request by kimwnasptd**
This PR makes a first try to create the `Getting Started` tutorial for the Data Science Stack.

It aims to explain:
1. How to install microK8s
2. How to setup microK8s to work with NVIDA drivers
3. How to download the dss cli snap
4. How to initialise the DSS
5. How to create a Notebook

### CODE REVIEW

File: docs/tutorial/installation.rst [hunk 1]
```diff
@@ -1,4 +1,114 @@
...
+All the workloads and state managed by DSS will be running on top of
+MicroK8s.
+
+You can install MicroK8s with the following commands:
+
+.. code-block:: bash
+
+   sudo snap install microk8s --channel 1.28/stable --classic
+   microk8s enable storage dns rbac
+   microk8s config > ./kubeconfig
```
Comment (misohu) [inline docs/tutorial/installation.rst:30]: Lets remove this one ... after merging @ca-scribner PR [here](https://github.com/canonical/data-science-stack/pull/113), we no longer need this instead we can just put to the step bellow: 

```
dss initialize --kubeconfig="$(microk8s kubeconfig)"
```

**Review by misohu — COMMENTED** [review]
Good job @kimwnasptd ... added some comments.

File: docs/tutorial/installation.rst [hunk 2]
```diff
@@ -1,4 +1,114 @@
-Installation Guide
-========================================
+Getting Started
+===============
 
-Step by step guide on how to get started with setting up DSS.
+What you'll need
```
[the remark this answers is not in the document]
Comment (kimwnasptd) [reply docs/tutorial/installation.rst:4]: Done! Added a bit of info as well before the prerequisites on what's the goal of the tutorial

File: docs/tutorial/installation.rst [hunk 3]
```diff
@@ -1,4 +1,114 @@
...
+^^^^^^^^^^^^^^^^
+
+* Ubuntu 22.04
+* Snapcraft (included in Ubuntu)
```
[the remark this answers is not in the document]
Comment (kimwnasptd) [reply docs/tutorial/installation.rst]: Removed Snapcraft

File: docs/tutorial/installation.rst [hunk 4]
```diff
@@ -1,4 +1,114 @@
...
+
+Under the hood this will create a K8s namespace and prepare the namespace
+accordingly, by installing MLFlow and setting up the shared storage for the
+Notebooks that will be launched in the future.
+
+You can initialise the DSS with the following commands:
+
+.. code-block:: bash
+
+   dss initialize
```
[the remark this answers is not in the document]
Comment (kimwnasptd) [reply docs/tutorial/installation.rst]: done

File: docs/tutorial/installation.rst [hunk 5]
```diff
@@ -1,4 +1,114 @@
...
+
+Install the DSS CLI
+^^^^^^^^^^^^^^^^^^^
+
+At this point we've installed MicroK8s and configured it to use the host's
+NVIDIA GPU.
+
+The next step now is to install the DSS CLI snap, which will be used for
+initialising the DSS on top of MicroK8s and managing user Notebooks and
+MLFlow.
```
[the remark this answers is not in the document]
Comment (kimwnasptd) [reply docs/tutorial/installation.rst]: Done!

File: docs/tutorial/installation.rst [hunk 6]
```diff
@@ -1,4 +1,114 @@
...
+
+To enable the NVIDIA runtimes on MicroK8s you need to run the following
+command:
+
+.. code-block:: bash
+
+   microk8s enable gpu
+
+
+Note that you can also instruct MicroK8s and the NVIDIA Operator to use
```
[the remark this answers is not in the document]
Comment (kimwnasptd) [reply docs/tutorial/installation.rst]: ACK! Rephrased a bit so that it's clear how the operator is doing this, PTAL

**Comment by kimwnasptd** [comment]
@misohu I addressed your comments, and left 2 open in case you want to take one more look on the changes I did on those

**Review by misohu — APPROVED** [review]
Good job @kimwnasptd

**Review by misohu — APPROVED** [review]

**Pull request merged**